.github/workflows/publish.yaml

```diff
@@ -0,0 +1,68 @@
+name: Publish App
+
+on:
+  push:
+    tags: [ 'v*' ]
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Setup Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Set variable for version
+        id: variable
+        run: echo "::set-output name=AppVersion::${GITHUB_REF//refs\/tags\//}"
```

Comment:
Should just set the env var so all gradle tasks pick it up automatically, like so:

https://github.com/jellyfin/jellyfin-sdk-kotlin/blob/5d1d7e3600d8f9d2f98dd5dbc2a68f4090f4f416/.github/workflows/sdk-publish.yaml#L22-L25

Reply:
Thy for the reference and given the different application than I assumed I agree that the env var will work better.
I used this method originally to make the use of the version more transparent (like we discussed in the test PR ages ago :sweat_smile:).

Reply:
Allow me to ask this though: Why do you need the `tr / -` pipe in the sdk repo?

Reply:
The `tr` part is not necessary but I kept it just to be safe IIRC

Reply:
mind if I omit it? additionally should I too remove the `v` prefix or let the VersionUtils.kt handle it?

Reply:
I'd rather keep it in case there was a reason that I forgot :p

Also the v prefix can be used to detect version tags in the CI but it should **always** pass the exact tag to the Gradle build scripts. The Gradle build scripts will handle it.

Reply:
Okay then I'll use your version of the export 1:1. Regarding the tag I agree, however was unsure since the old script handled only the `refs/tags` bit so wanted to be sure :+1: